test/metabase/models/setting_test.clj

```diff
@@ -199,6 +199,30 @@
               #"Cannot initialize setting before the db is set up"
               (setting/get :test-setting-custom-init)))))))
 
+(def ^:private base-options
+  {:setter   :none
+   :type     :csv
+   :default "base-default"})
+
+(defsetting test-base-setting
+  "Setting to test the `:feature` property of settings. This only shows up in dev."
+    :visibility :internal
+    :default "my,default"
+    :base base-options)
+
+(deftest ^:parallel defsetting-with-base-test
+  (testing "A setting which specifies some base options"
+    (let [setting (setting/resolve-setting :test-base-setting)]
+      (testing "Uses non-overridden base options"
+        (is (= :csv (:type setting)))
+        (is (= :none (:setter setting)))
+        ;; Obfuscate test-base-setting! to avoid a false positive from `deftest-check-parallel`
```

Comment:
a better way to do this is to tell kondo to ignore it.

```
#_{:clj-kondo/ignore [linter-name]}
```

Reply:
I think this only works when the linter reports a finding, but in this case the hook throws an exception. I'll give it a try though.

Reply:
what's the key of that error? I'll see if I could fix it, looks like we need to update the hook to respect kondo/ignore

Reply:
Ah I took another look and see that `error!` is just an alias for `reg-finding!`, which encapsulates the key `:metabase/validate-deftest`.

I'd thought Cursive would have shown me the key if there was one, and that `error!` wasn't sending one. Cool, I've pushed a version with the ignore. Unfortunately it seems that I have to ignore the entire test, which isn't ideal. I might actually prefer the symbol-builder approach if we need to extend this test in future.

Reply:
Score one for Cider! I see that shows the ignore key front and center 😄